jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties

```diff
@@ -25,36 +25,19 @@ LetClassExtend = Let ''{0}'' extend ''{1}''
 # FilterImplementationQuickFix
 LetClassImplement = Let ''{0}'' implement ''{1}''
 
-# InsertAnnotationAttributesQuickFix
-AddAtoB = Add {0} to {1}
-
-# InsertAnnotationMissingQuickFix
-InsertItem = Insert {0}
-
-# RemoveAnnotationConflictQuickFix
-RemoveItem = Remove {0}
-
-# RemoveParamAnnotationQuickFix
+# RemoveMethodParamAnnotationQuickFix
 RemoveTheModifierFromParameter = Remove the {0} modifier from parameter ''{1}''
 
-# RemoveModifierConflictQuickFix
-RemoveTheModifierFromThisVariable = Remove the ''{0}'' modifier from this variable
-RemoveTheModifierFromThisField = Remove the ''{0}'' modifier from this field
-RemoveTheModifierFromThisMethod = Remove the ''{0}'' modifier from this method
-RemoveTheModifierFromThisClass = Remove the ''{0}'' modifier from this class
-RemoveTheModifierFromThis = Remove the ''{0}'' modifier from this {1}
-
-# RemoveMethodParametersQuickFix
+# RemoveAllMethodParametersQuickFix
```

Comment:
1. I see a lot of messages are removed. I hope this was on purpose.
2. Would it make sense to change the path of the messages.properties to match the path of Messages.java?